Cloud Enterprise/Getting Started Examples/aws/terraform/variables.tf

```diff
@@ -2,87 +2,89 @@
 #### Variables to configure (or be prompted about) ####
 #######################################################
 
-# Your aws profile name
-variable "profile" {
-  description = "Your aws profile"
-}
-
 # Your IP address, so that you will be whitelisted against security groups
 variable "trusted_network" {
-  description = "CIDR formatted IP (<IP Address>/32) or network that will be allowed access (you can use 0.0.0.0/0 for unrestricted access)"
+  description = "CIDR formatted IP (<IP Address>/32) or Network that will be allowed administrative access"
```

Comment:
I'm not sure about the flow here, but could you pull the latest changes from my recent changes? I've updated the descriptions of some variables

Reply:
tiny tweak from your changes to make clear that the setting is used for SSH access.